src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -13,6 +13,39 @@ const MISSING_BLOCK_ERROR_MESSAGE = 'The "block" field type needs at least one t
 const BLOCK_PREVIEW_TAG = 'sulu.block_preview';
 
 export default class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
+    componentDidUpdate(prevProps: FieldTypeProps<Array<BlockEntry>>) {
+        const {defaultType, types, value: oldValues, onChange} = this.props;
+        const {types: oldTypes} = prevProps;
+
+        if (!oldValues) {
+            return;
```

Comment:
I would not use early returns in here. The problem is that this gets very complicated, if there is a second prop changing, that you want to handle.